app.py

```diff
@@ -551,13 +819,20 @@ async def send_chat_request(request):
 
     return response
 
+
 async def complete_chat_request(request_body):
-    response = await send_chat_request(request_body)
-    history_metadata = request_body.get("history_metadata", {})
+    if DATASOURCE_TYPE == "Promptflow":
+        response = await promptflow_request(request_body)
+        history_metadata = request_body.get("history_metadata", {})
+        return format_pf_non_streaming_response(response, history_metadata)
+    else:
+        response = await send_chat_request(request_body)
+        history_metadata = request_body.get("history_metadata", {})
+        return format_non_streaming_response(response, history_metadata)
 
-    return format_non_streaming_response(response, history_metadata)
 
 async def stream_chat_request(request_body):
```

Comment:
Is there a streaming equivalent for the promptflow endpoint call?  I'm just wondering if we should raise an error if STREAM = true with promptflow settings, or just let the request go to AOAI.